src/auth/auth.js

```diff
@@ -124,10 +125,11 @@ Auth.prototype.login = function(token) {
 	this.setUser(UserFromToken(token))
 
 	if (this.loggedIn) {
-		localStorage.setItem(TokenName, token)
+		sessionStorage.setItem(TokenName, token)
```

Comment:
call the function instead, or create a function for this
Auth.prototype.setToken()

Reply:
done